docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -426,6 +426,24 @@ ARRAY_UNION(ARRAY[1, 2, 3, 1, 2], [4, 1])  => [1, 2, 3, 4]
 ARRAY_UNION(ARRAY['apple', 'apple', NULL, 'cherry'], ARRAY['cherry'])  => ['apple', NULL, 'cherry']
 ```
 
+### `ARRAY_CONCAT`
+
+Since: 0.20.0
+
+```sql
+ARRAY_CONCAT(array1, array2)
+```
+
+Returns an array representing the concatenation of both input arrays.
+
+Returns NULL if both input arrays are NULL.
```

Comment:
```suggestion
Returns NULL if both input arrays are NULL. If only one argument is NULL, the result is the other argument.
```

... Or any other way you prefer to say it, but we should specify the behavior if just one arg is null.